membership_initial_fee/README.rst

```diff
@@ -0,0 +1,96 @@
+===========================
+Initial fee for memberships
+===========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fvertical--association-lightgray.png?logo=github
+    :target: https://github.com/OCA/vertical-association/tree/12.0/membership_initial_fee
+    :alt: OCA/vertical-association
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/vertical-association-12-0/vertical-association-12-0-membership_initial_fee
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/208/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+Charge an initial fee when a partner is invoiced for the first time with a
+member product.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+Define a member product, and select 'Fixed amount' or 'Percentage of the price'
+for invoicing an extra charge in the first invoice that is created with this
+member product.
+
+By default, a line with the description *Membership initial fee*. If you want
+to change this text, you can set a different sale description in the product
+used for the fee.
+
+Known issues / Roadmap
+======================
+
+* Add initial fee information to membership analysis.
+* Add some criteria for adding initial fee:
+   * Add initial fee if last membership ended x weeks/months/years ago
+   * Add initial fee if last membership product_id is different
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/vertical-association/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/vertical-association/issues/new?body=module:%20membership_initial_fee%0Aversion:%2012.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Tecnativa
+
+Contributors
+~~~~~~~~~~~~
+
+* `Tecnativa <https://www.tecnativa.com>`__:
+
+  * Pedro M. Baeza
+  * David Vidal
```

Comment:
Well after reviewing this modules from v8, v10, v11, v12..... I think this is contribution